docs/sources/reference/cli/run.md

```diff
@@ -55,6 +55,7 @@ The following flags are supported:
 * `--config.bypass-conversion-errors`: Enable bypassing errors when converting (default `false`).
 * `--config.extra-args`: Extra arguments from the original format used by the converter.
 * `--stability.level`: The minimum permitted stability level of functionality to run. Supported values: `experimental`, `public-preview`, `generally-available` (default `"generally-available"`).
+* `--community-component`: Enable community components (default `false`).
```

Comment:
I wonder if there's an existing convention for the flag names we could conform to here. For example, we have `--cluster.enabled` so `--community.enabled` or `--community-components.enabled` would align with that potentially.